client/landing/gutenboarding/components/plans/plans-grid/style.scss

```diff
@@ -0,0 +1,45 @@
+@import 'assets/stylesheets/gutenberg-base-styles';
+@import '../../../mixins.scss';
+
+.plans-grid {
+	// Space to accomodate sticky footer
+	margin-bottom: 85px; // @TODO: replace with a variable
+
+	@include break-small {
+		margin-bottom: 0;
+	}
+}
+
+.plans-grid__header { // @TODO: replace with a generic component
+	@include onboarding-heading-padding;
+	display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+}
+
+// @TODO: remove once the real grid is implemented
```

Comment:
I'd be fine if there would be even just empty canvas for starters, but ok with this too if you prefer.